### DISCUSSION TITLE
nfsen on sourceforge

### DISCUSSION THREAD

**Discussion by VersusBG** [phaag/nfsen#14] [Q&A] [answered]
Is there a plan of updating the nfsen version in sourceforge or this version on github will be the main source for future updates/versions ?

**Comment by phaag** [comment, marked as answer]
There are no plans to update NfSen on Sourceforge. Maybe the doku will move as well. The repo here at Github received some patches to work with latest nfdump. It will not get new features. The code is too old and need to be rewritten. There may be an other new NfSen in future but that will ebnd up here in Github too.
